src/main/java/Dao/AddressDao.java

```diff
@@ -0,0 +1,96 @@
+package Dao;
+
+import Entities.Address;
+
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class AddressDao extends AbstractDao<Address> {
+    @Override
+    public Address getById(int id) {
+        String SQL_SELECT = "Select * from address where address_id=?";
+        try (PreparedStatement preparedStatement = this.executePrepareStatement(SQL_SELECT)) {

```

Comment:
Instead of try-catch , you can use try with resources to close the connection after use.

Reply:
I agree

Reply:
Done!